dataline-scheduler/src/main/java/io/dataline/scheduler/ScopeHelper.java

```diff
@@ -31,10 +31,14 @@
 
   private static final String SCOPE_DELIMITER = ":";
 
+  public static String getScopePrefix(JobConfig.ConfigType configType) {
```

Comment:
i know this is a dorky method, but i was looking at the scopes in the db and was wondering if we were being careful about capitalization. at least now there's a standard (even if it is the default behavior).